libs/wingc/src/parser.rs

```diff
@@ -977,8 +977,41 @@ impl<'s> Parser<'s> {
 			};
 		}
 
+		if WINGSDK_BRINGABLE_MODULES.contains(&module_name.name.as_str()) || module_name.name == WINGSDK_STD_MODULE {
+			return Ok(StmtKind::Bring {
+				source: BringSource::BuiltinModule(module_name),
+				identifier: alias,
+			});
+		}
+
+		// check if a trusted library exists with this name
+		let source_dir = Utf8Path::new(&self.source_name).parent().unwrap();
+		let module_dir =
+			wingii::util::package_json::find_dependency_directory(&format!("@winglibs/{}", module_name.name), &source_dir)
```

Comment:
please put the string `"@winglibs"` in a const and use everywhere so we have a single location where it's codified.